.github/CODEOWNERS

```diff
@@ -1,106 +1,586 @@
+
+# ServiceLabel: %Reservations %Service Attention
+/sdk/reservations/Microsoft.Azure.Management.Reservations/            @corquiri
+
+# ServiceLabel: %Resource Authorization %Service Attention
+#/<NotInRepo>/            @darshanhs90 @AshishGargMicrosoft
+
+# ServiceLabel: %Resource Graph %Service Attention
+/sdk/resourcegraph/Microsoft.Azure.Management.ResourceGraph/            @chiragg4u
+
+# ServiceLabel: %Resource Health %Service Attention
+#/<NotInRepo>/            @stephbaron
+
+# ServiceLabel: %Scheduler %Service Attention
+/sdk/scheduler/Microsoft.Azure.Management.Scheduler/            @derek1ee
+
+# ServiceLabel: %Security %Service Attention
+#/<NotInRepo>/            @chlahav
 
 # PRLabel: %Search
 /sdk/search/                                    @brjohnstmsft @arv100kri @bleroy @tg-msft @heaths
-/sdk/search/Microsoft.*/                        @brjohnstmsft @arv100kri @bleroy
 
-# PRLabel: %Service Bus
+# ServiceLabel: %Search %Service Attention
+/sdk/search/Microsoft.*/                        @brjohnstmsft @arv100kri @bleroy @tjacobhi @markheff @miwelsh
+
+# PRLabel: %Service Bus (Client)
 /sdk/servicebus/                                @JoshLove-msft @ShivangiReja @jsquire @MiYanni
-/sdk/servicebus/Microsoft.*/                    @nemakam
+
+# ServiceLabel: %Service Bus %Service Attention
+/sdk/servicebus/Microsoft.*/                    @axisc
+
+# ServiceLabel: %Service Fabric %Service Attention
+/sdk/servicefabric/Microsoft.Azure.Management.ServiceFabric/           @QingChenmsft @vaishnavk @juhacket
+
+# ServiceLabel: %Schema Registry %Service Attention
+/sdk/schemaregistry/            @arerlend @alzimmermsft
+
+# ServiceLabel: %SignalR %Service Attention
+/sdk/signalr/Microsoft.Azure.Management.SignalR/            @sffamily @chenkennt
+
+# ServiceLabel: %SQL %Service Attention
+#/<NotInRepo>/            @azureSQLGitHub
+
+# ServiceLabel: %SQL - VM %Service Attention
+/sdk/sqlvirtualmachine/Microsoft.Azure.Management.SqlVirtualMachine/            @azureSQLGitHub
+
+# ServiceLabel: %SQL - Backup & Restore %Service Attention
+#/<NotInRepo>/            @azureSQLGitHub
+
+# ServiceLabel: %SQL - Data Security %Service Attention
+#/<NotInRepo>/            @azureSQLGitHub
+
+# ServiceLabel: %SQL - Elastic Jobs %Service Attention
+#/<NotInRepo>/            @azureSQLGitHub
+
+# ServiceLabel: %SQL - Managed Instance %Service Attention
+#/<NotInRepo>/            @azureSQLGitHub
+
+# ServiceLabel: %SQL - Replication & Failover %Service Attention
+#/<NotInRepo>/            @azureSQLGitHub
 
 # PRLabel: %Storage
 /sdk/storage/                                   @tg-msft
-/sdk/storage/Azure.Storage.*/                   @amishra-dev @seanmcc-msft @amnguye @kasobol-msft @tg-msft
 
-# PRLabel: %Tables
-/sdk/tables/                                    @christothes
+# ServiceLabel: %Storage %Service Attention
+/sdk/storage/Azure.Storage.*/                   @amishra-dev @seanmcc-msft @amnguye @kasobol-msft @tg-msft @xgithubtriage
```

Comment:
what is xgithubtriage going to do with storage PRs?

btw. please make sure this https://github.com/Azure/azure-sdk-for-net/pull/16200 gets reflected here correctly.

Reply:
https://github.com/Azure/azure-sdk-for-net/pull/16363 as well.

Reply:
@kasobol-msft what should we have instead of xgithubtriage? This was in the internal wiki page.

Reply:
@kasobol-msft I pulled in those 2 PRs.